easybuild/tools/module_generator.py

```diff
@@ -671,7 +656,7 @@ def _generate_help_text(self):
             lines.extend(self._generate_section("Compatible modules", compatible_modules_txt))
 
         # Extensions (if any)
-        extensions = self._generate_extension_list()
+        extensions = self.app.make_extension_string()
```

Comment:
the line below uses a join on this var which is now a string instead of a list? It also needs to be adjusted?

Reply:
I had to stare at this for a bit now but it is correct as-is:
- `_generate_extension_list` did [return a string](https://github.com/easybuilders/easybuild-framework/pull/4690/files#diff-4af72e9777d353325a29df400ab4229e14defd653e413db51786dc2172d3baefL619), so behavior is unchanged
- The `join` happens on the result of `wrap` which takes a string and returns a list of strings

Reply:
aha, I was kinda surprised this didn't trigger a failed test.